dev/gpctl/cmd/debug-log.go

```diff
@@ -73,6 +70,29 @@ var debugLogCmd = &cobra.Command{
 	},
 }
 
+func enableLogDebug(podName string, cfg *rest.Config, namespace string) error {
+	freePort, err := GetFreePort()
+	if err != nil {
+		return err
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	port := fmt.Sprintf("%d:6060", freePort)
+	readychan, errchan := util.ForwardPort(ctx, cfg, namespace, podName, port)
+	select {
+	case <-readychan:
+	case err := <-errchan:
+		return err
+	case <-ctx.Done():
+		return ctx.Err()
+	}
+
+	_, err = http.Post(fmt.Sprintf("http://localhost:%d/debug/logging", freePort), "", bytes.NewReader([]byte(`{"level":"debug"}`)))
```

Comment:
If we add `Content-Type: application/json` this will also work for TS components. :+1:

Reply:
Will add

Reply:
Done